wcfsetup/install/files/lib/system/user/notification/UserNotificationHandler.class.php

```diff
@@ -924,6 +925,7 @@ public function markAsConfirmed($eventName, $objectType, array $recipientIDs, ar
         $parameters = $conditions->getParameters();
         \array_unshift($parameters, TIME_NOW);
         $statement->execute($parameters);
+        $affected = $statement->getAffectedRows();
```

Comment:
```suggestion
        $confirmedCount = $statement->getAffectedRows();
```

is a more descriptive name (same below).